examples/all-clusters-app/all-clusters-common/src/concentration-measurement-instances.cpp

```diff
@@ -0,0 +1,239 @@
+    {
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer =
+            new Instance(endpoint, TotalVolatileOrganicCompoundsConcentrationMeasurement::Id, MeasurementMediumEnum::kAir,
+                         MeasurementUnitEnum::kPpm, gConcentrationMeasurementAllFeatures.Raw());
+
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetMinMeasuredValue(DataModel::Nullable<float>(0.0));
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetMaxMeasuredValue(DataModel::Nullable<float>(1000.0));
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetMeasuredValue(DataModel::Nullable<float>(1.0));
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetPeakMeasuredValue(DataModel::Nullable<float>(1.0));
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetPeakMeasuredValueWindow(320);
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetAverageMeasuredValue(DataModel::Nullable<float>(1.0));
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetAverageMeasuredValueWindow(320);
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetUncertainty(0.0);
+        gTotalVolatileOrganicCompoundsConcentrationMeasurementServer->SetLevel(LevelValueEnum::kLow);
+    }
+}
+
+void emberAfOzoneConcentrationMeasurementClusterInitCallback(chip::EndpointId endpoint)
+{
+    if (gOzoneConcentrationMeasurementServer == nullptr)
+    {
+        gOzoneConcentrationMeasurementServer =
+            new Instance(endpoint, OzoneConcentrationMeasurement::Id, MeasurementMediumEnum::kAir, MeasurementUnitEnum::kPpm,
+                         gConcentrationMeasurementAllFeatures.Raw());
+
+        gOzoneConcentrationMeasurementServer->SetMinMeasuredValue(DataModel::Nullable<float>(0.0));
+        gOzoneConcentrationMeasurementServer->SetMaxMeasuredValue(DataModel::Nullable<float>(1000.0));
+        gOzoneConcentrationMeasurementServer->SetMeasuredValue(DataModel::Nullable<float>(1.0));
+        gOzoneConcentrationMeasurementServer->SetPeakMeasuredValue(DataModel::Nullable<float>(1.0));
+        gOzoneConcentrationMeasurementServer->SetPeakMeasuredValueWindow(320);
+        gOzoneConcentrationMeasurementServer->SetAverageMeasuredValue(DataModel::Nullable<float>(1.0));
+        gOzoneConcentrationMeasurementServer->SetAverageMeasuredValueWindow(320);
+        gOzoneConcentrationMeasurementServer->SetUncertainty(0.0);
+        gOzoneConcentrationMeasurementServer->SetLevel(LevelValueEnum::kLow);
+    }
+}
+
+void emberAfFormaldehydeConcentrationMeasurementClusterInitCallback(chip::EndpointId endpoint)
+{
+    if (gFormaldehydeConcentrationMeasurementServer == nullptr)
+    {
+        gFormaldehydeConcentrationMeasurementServer =
+            new Instance(endpoint, FormaldehydeConcentrationMeasurement::Id, MeasurementMediumEnum::kAir, MeasurementUnitEnum::kPpm,
```

Comment:
So where are these allocations cleaned up?  Somewhere in shutdown we should `delete` these things, right?